src/components/Sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,89 @@
+import { Link } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import { ChakraProvider, Button, VStack, Divider, Icon, Flex, Image, Box, Text, HStack } from '@chakra-ui/react';
+import 'boxicons';
+import fphLogo from './fph-logo.png.png';
+
+
+function Sidebar({ isAdmin }) {
+  const businessList = [
+    { name: 'Home', path: '/BusinessDashboard', icon: 'home-smile'},
+    { name: 'Donation History', path: '/BusinessDonationTrackingForm', icon: 'heart-circle' },
+    { name: 'Supply Request', path: '/ContactUs', icon: 'package'},
+    { name: 'Settings', path: '/EditContactInformation', icon: 'cog' },
+  ];
+
+  // { name: 'Business Onboarding', path: '/BusinessOnboardingForm', icon: 'edit-alt' },
+  // { name: 'Business Notification Center', path: '/BusinessNotificationCenter', icon: 'bell' },
+
+  const adminList = [
+    { name: 'Home', path: '/AdminDashboard', icon: 'home-smile'},
+    { name: 'Donation Tracking', path: '/AdminDashboard', icon: 'building-house'},
+    { name: 'Donation Items', path: '/AdminDashboard', icon: 'heart-circle'},
+    { name: 'Settings', path: '/EditContactInformation', icon: 'cog'},
+  ];
+  // { name: 'All Businesses', path: '/AdminAllBusinesses', icon: 'heart-circle'},
+
+  // { name: 'Manage Forms', path: '/AdminManageForms', icon: 'edit'},
+  // { name: 'Team Management', path: '/AdminTeamManagement', icon: 'group'},
+
+  const navList = isAdmin ? adminList : businessList;
+  // const navList = businessList;
+  return (
+    <ChakraProvider>
+    <>
+      <Flex width='40vh'>
+        <VStack>
+          <HStack marginLeft='1vh' marginTop='1vh' >
+            <Image
+              borderRadius='full'
+              boxSize='7.5vh'
+              src={fphLogo}
+              alt='jit'
```

Comment:
replace jit with a legitimate alt text because this could easily slip through later and we don't want someone using a screen reader to have this read as jit later